java/alizer-api/src/main/java/com/redhat/devtools/alizer/api/LanguageFileHandler.java

```diff
@@ -49,7 +51,7 @@ public static LanguageFileHandler get() {
 
     private void initLanguages() {
         try {
-            String yamlAsString = IOUtils.toString(LanguageFileHandler.class.getResourceAsStream("/" + LANGUAGES_YAML_PATH), Charset.defaultCharset());
+            String yamlAsString = IOUtils.toString(new File(LANGUAGES_YAML_PATH).toURI(), Charset.defaultCharset());
```

Comment:
That will request every user of the library to have that file on the file system, so that is not scalable.
The solution is to keep loading the file from the class path and modify the build (pom.xml) so that the common file is copied to the output folder before the compilation is performed